### PR TITLE
Fix examples template

### PR DESCRIPTION
**Pull request by liyokuna**
Changes:
- Removing old swiper code in homepage in Examples.
- Unsetting the position of the pagination and reduicing the height of the custom carousel

### CODE REVIEW

File: site/docs/4.3/examples/orange-homepage/index.html [hunk 1]
```diff
@@ -74,69 +74,45 @@
...
-          <div class="swiper-slide">
-            <img src="./images/offset_LARGE_118551_Parties_foncees_du_bleu_de_travail_en_jaune.jpg" alt="" class="img-fluid" />
-            <div class="frame">
-              <img src="./images/Fun-frame_frame-carousel.png" alt="Fun" class="img-fluid" />
-            </div>
-            <div class="caption ">
-              <div class="title ">You can use any hero picture to illustrate your purpose insert your body text in here. Ommoditatur sendand
-                amusanti nobisci psandae dolupta tatur, con corrum sam fugitatiunt aliae pa doluptatur sit aut alite excerei
-                ctasimin. </div>
+          <div class="swiper-slide" id="slide2">
```
Comment (Lausselloic) [inline site/docs/4.3/examples/orange-homepage/index.html]: duplicate ID

File: site/docs/4.3/examples/orange-homepage/index.html [hunk 2]
```diff
@@ -193,39 +169,49 @@ <h2>Discovery module – two pop-outs in a carousel Title 1 here</h2>
...
+                        pa doluptatur sit aut alite excerei ctasimin.
+                      </p>
+                      <a href="#" class="btn btn-secondary discovery-header-button">Find out more<span class="sr-only">&nbsp;about the TITLE OF THE INSIGHT</span></a>
                 </div>
               </div>
             </div>
           </div>
 
-          <div class="swiper-slide">
+          <div class="swiper-slide" id="slide2">
```
Comment (Lausselloic) [inline site/docs/4.3/examples/orange-homepage/index.html]: second occurrence

File: site/docs/4.3/examples/orange-homepage/index.html [hunk 3]
```diff
@@ -74,69 +74,45 @@
...
-                    <img src="./images/offset_LARGE_118551_Parties_foncees_du_bleu_de_travail_en_jaune.jpg" alt=""  class="img-fluid"/>
-                    <div class="frame">
-                        <img src="./images/Fun-frame_frame-carousel.png" alt="Fun"  class="img-fluid"/>
-                    </div>
-                    <div class="caption ">
-                        <div class="title ">You can use any hero picture to illustrate your purpose insert your body text in here. Ommoditatur sendand amusanti nobisci psandae dolupta tatur, con corrum sam fugitatiunt aliae pa doluptatur sit aut alite excerei ctasimin. </div>
-                    </div>
+      <div id="main-carousel" class="swiper-container" style="width: 37.5rem;">
+        <div class="swiper-wrapper">
+          <div class="swiper-slide" id="slide1">
```
Comment (Lausselloic) [inline site/docs/4.3/examples/orange-homepage/index.html]: duplicateID

File: site/docs/4.3/examples/orange-homepage/index.html [hunk 4]
```diff
@@ -182,9 +158,9 @@ <h2>Discovery module – one pop-out Title here</h2>
     <!-- Discovery module – two or more pop-outs in a carousel -->
     <!-- According to the brand, frames and popout can't be bigger than 220px width-->
     <div class="container-fluid">
-      <div id="discovery-carousel" class="swiper-container">
+      <div id="discovery-carousel" class="swiper-container" style="width: 37.5rem;">
         <div class="swiper-wrapper">
-          <div class="swiper-slide">
+          <div class="swiper-slide" id="slide1">
```
Comment (Lausselloic) [inline site/docs/4.3/examples/orange-homepage/index.html]: duplicateID

File: scss/_o-carousel.scss [hunk 5]
```diff
@@ -135,13 +135,12 @@
 }
 
 .swiper-wrapper {
-  margin-bottom: 30px;
+  margin-bottom: 10px;
```
[the remark this answers is not in the document]
Comment (liyokuna) [reply scss/_o-carousel.scss]: There was too much space between the bottom of the carousel and the indicators.

File: site/docs/4.3/examples/orange-homepage/index.html (hunk 4 above)
Comment (Lausselloic) [reply site/docs/4.3/examples/orange-homepage/index.html]: fixed

File: site/docs/4.3/examples/orange-homepage/index.html (hunk 3 above)
Comment (Lausselloic) [reply site/docs/4.3/examples/orange-homepage/index.html]: fixed

File: scss/_o-carousel.scss [hunk 6]
```diff
@@ -135,13 +135,12 @@
...
 }
 
 // Boosted mod
 .swiper-pagination {
-  position: absolute !important; // stylelint-disable-line declaration-no-important
-  bottom: 50px !important; // stylelint-disable-line declaration-no-important
+  position: unset !important; // stylelint-disable-line declaration-no-important
```
[the remark this answers is not in the document]
Comment (liyokuna) [reply scss/_o-carousel.scss]: If you add the buton pause and play, the indicators are above the button and it makes hard to click.
I should have test with z-index...

File: site/docs/4.3/examples/orange-homepage/index.html (hunk 2 above)
Comment (Lausselloic) [reply site/docs/4.3/examples/orange-homepage/index.html]: fixed

File: site/docs/4.3/examples/orange-homepage/index.html (hunk 1 above)
Comment (Lausselloic) [reply site/docs/4.3/examples/orange-homepage/index.html]: fixed

File: site/docs/4.3/examples/orange-homepage/index.html [hunk 7]
```diff
@@ -74,69 +74,45 @@
...
-                </div>
-                <div>
-                    <img src="./images/offset_LARGE_118551_Parties_foncees_du_bleu_de_travail_en_jaune.jpg" alt=""  class="img-fluid"/>
-                    <div class="frame">
-                        <img src="./images/Fun-frame_frame-carousel.png" alt="Fun"  class="img-fluid"/>
-                    </div>
-                    <div class="caption ">
-                        <div class="title ">You can use any hero picture to illustrate your purpose insert your body text in here. Ommoditatur sendand amusanti nobisci psandae dolupta tatur, con corrum sam fugitatiunt aliae pa doluptatur sit aut alite excerei ctasimin. </div>
-                    </div>
+      <div id="main-carousel" class="swiper-container" style="width: 37.5rem;">
```
[the remark this answers is not in the document]
Comment (Lausselloic) [reply site/docs/4.3/examples/orange-homepage/index.html]: fixed

File: site/docs/4.3/examples/orange-homepage/index.html (hunk 1 above)
Comment (liyokuna) [reply site/docs/4.3/examples/orange-homepage/index.html]: ohhh noooo.

**Pull request merged**